src/components/cuda_pw/cuda_common.c

```diff
@@ -0,0 +1,168 @@
+#include <stdio.h>
+#include <dlfcn.h>
+#include <papi.h>
+
+#include "cuda_common.h"
+
+int get_env_PAPI_CUDA_ROOT(void)
```

Comment:
Don't mix upper and lower case. Be consistent. We use lower case for function names.